### PR TITLE
#161 Remove pycache directories files on build

### LINKED ISSUE #161: Create __pycache__ removal script
## Overview
Old `__pycache__` files can interfere with proper running of `docker-compose -f local.yml build` and may even cause the following error: 

```python
import file mismatch:
imported module 'app.src.tests.test_load_ct' has this __file__ attribute:
  /Volumes/Nahla/alcf/prediction/src/tests/test_load_ct.py
which is not the same as the test file we want to collect:
  /app/src/tests/test_load_ct.py
HINT: remove __pycache__ / .pyc files and/or use a unique basename for your test file modules
``` 

There should be a convenient script that can be run to remove all `__pycache__` files.

## Expected Behavior
Running the script should remove all `__pycache__` files.

## Technical details

- [Relevant helper script](https://github.com/reubano/meza/blob/master/helpers/clean)

- This feature should be implemented in [`compose/prediction/Dockerfile-dev`](https://github.com/concept-to-clinic/concept-to-clinic/tree/master/compose/prediction/Dockerfile-dev)

## Acceptance criteria

 - [ ] running `docker-compose -f local.yml build prediction` should first remove previously existing `__pycache__` files 

*NOTE: All PRs must follow the standard PR checklist.*

### PR DESCRIPTION
**Pull request by rahit**
Remove all __pycache__ files when you run `docker-compose -f local.yml build prediction`.

## Description

Added a one-line code in Docker file referenced at #161  by @reubano 

## Reference to official issue

fixes #161 

## Motivation and Context

Old __pycache__ files can interfere with proper running of docker-compose -f local.yml build and may even cause the following error:
```
import file mismatch:
imported module 'app.src.tests.test_load_ct' has this __file__ attribute:
  /Volumes/Nahla/alcf/prediction/src/tests/test_load_ct.py
which is not the same as the test file we want to collect:
  /app/src/tests/test_load_ct.py
```
There should be a convenient script that can be run to remove all __pycache__ files.

## How Has This Been Tested?

Run build several times without any error.

## CLA
- [x] I have signed the CLA; if other committers are in the commit history, they have signed the CLA as well

### CODE REVIEW

File: compose/prediction/Dockerfile-dev [hunk 1]
```diff
@@ -1,3 +1,3 @@
 FROM concepttoclinic_base
 
-WORKDIR /app
+WORKDIR /app
```
Comment (lamby) [inline compose/prediction/Dockerfile-dev]: Please take care not to make unnecessary changes  - makes it harder to review!

**Comment by lamby** [comment]
LGTM, but I'm seeing 18 commits on this pull request. Can you rebase? :)

**Comment by rahit** [comment]
@lamby  done

**Comment by lamby** [comment]
merged, many thanks
